README.md

```diff
@@ -578,6 +600,21 @@ source dbt_packages/fivetran_utils/generate_columns.sh '../dbt_apple_search_ads_
 In that example, it will:
 * Create a `get_campaign_history_columns.sql` file in the `macros` directory, with the necessary macro within it.
 
+----
+
+### generate_docs.sh([source](generate_docs.sh))
+
+This bash file can be used to create or replace package documentation (`<project_name>/docs`). 
```

Comment:
It isn't clear where I need to run this command. Do I run it in the direction of the `integration_tests` folder, or within the root of my `development` dbt project? Can we specify that here.

Reply:
Sure thing! Where is your development folder located wrt the dbt_jira_source directory?

Reply:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/94874400/155804266-71284942-9fab-4f0e-bf2f-46e9c6294460.png">
This macro is designed to work in all directories under `stripe` (the project directory) that exist at the same level as `dbt_stripe`. So you can run this from `dev` which exists adjacent to `dbt_stripe`. I hope that helps! If it still doesn't work let me know. 

@fivetran-joemarkiewicz